CHANGELOG.md

```diff
@@ -65,6 +65,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Added `on_transcription_stopped` and `on_transcription_error` to Daily
   callbacks.
 
+- Added `source` field to `ErrorFrame` to indicate `FrameProcessor` that generated the error.
```

Comment:
I think you need to rebase and add this line to a new `### Added` section under `[Unreleased]`.